apps/configurator/lib/configurator_web/controllers/skill_html/index.html.heex

```diff
@@ -0,0 +1,65 @@
+<.header>
+  Listing Config skills
+  <:actions>
+    <.link href={~p"/skills/new"}>
+      <.button>New Skill</.button>
+    </.link>
+  </:actions>
+</.header>
+
+<.table id="skills" rows={@skills} row_click={&JS.navigate(~p"/skills/#{&1}")}>
+  <:col :let={skill} label="Name"><%= skill.name %></:col>
+  <:col :let={skill} label="Activation delay ms"><%= skill.activation_delay_ms %></:col>
+  <:col :let={skill} label="Autoaim"><%= skill.autoaim %></:col>
+  <:col :let={skill} label="Block movement"><%= skill.block_movement %></:col>
+  <:col :let={skill} label="Can pick destination"><%= skill.can_pick_destination %></:col>
+  <:col :let={skill} label="Cooldown mechanism"><%= skill.cooldown_mechanism %></:col>
+  <:col :let={skill} label="Cooldown ms"><%= skill.cooldown_ms %></:col>
+  <:col :let={skill} label="Execution duration ms"><%= skill.execution_duration_ms %></:col>
+  <:col :let={skill} label="Inmune while executing"><%= skill.inmune_while_executing %></:col>
+  <:col :let={skill} label="Is passive"><%= skill.is_passive %></:col>
+  <:col :let={skill} label="Max autoaim range"><%= skill.max_autoaim_range %></:col>
+  <:col :let={skill} label="Stamina cost"><%= skill.stamina_cost %></:col>
+  <:action :let={skill}>
+    <.button type="button" phx-click={show_modal("skill-mechanics-#{skill.id}")}>Mechanics</.button>
+    <.modal id={"skill-mechanics-#{skill.id}"}>
```

Comment:
Maybe this modal show have some submit button?

Reply:
I prefer to merge it as it is and improve the UX later.

Reply:
sounds good